src/components/CurrencyInput.tsx

```diff
@@ -104,9 +104,9 @@ export const CurrencyInput: FC<CurrencyInputProps> = forwardRef<
     };
 
     const formattedStateValue =
-      defaultValue !== undefined && defaultValue !== null
+      defaultValue != null
```

Comment:
This is just a clean-up. The double-equal null logic is equivalent to checking for `null` and `undefined`. It makes the code more concise.

Reply:
Interesting, I did not know that!

Although, I personally would prefer to be explicit rather than concise, so I was tempted to say keep it as is, but since all the tests passed anyway, I thought that's good enough for me :D